components/Button.js

```diff
@@ -10,54 +10,104 @@ export function Button({ type, variant, onClick, children }) {
 
 const DefaultButton = styled.button`
   border: none;
+  border-radius: 3px;
   box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
     rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
-  border-radius: 3px;
-  background-color: oldlace;
+  background-color: var(--fourth-color);
+  font-size: 16px;
```

Comment:
For accessibility reasons, I suggest to use the unit `rem` for `font-size` instead of `px`. Have a look at [this article](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#accessibility-considerations).

```suggestion
  font-size: 1rem;
```